### PR TITLE
Remove egicli and refresh token references

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
Some changed in `fedcloud` cli are needed before merging this one. So keeping it in draft for the time being

**Review by gwarf — APPROVED** [review]
Cool to see this!
👍

**Comment by enolfc** [comment]
@andrea-manzi can you also take a look
@tdviet please also check that things make sense

**Comment by andrea-manzi** [comment]
yup sorry, i will also do some tests

File: content/en/providers/notebooks/operations/_index.md [hunk 1]
```diff
@@ -52,39 +52,25 @@ cd <new provider>/terraform
...
 ```shell
-$ egicli endpoint projects --site CESGA
-id                                Name                 enabled    site
---------------------------------  -------------------  ---------  ------
-3a8e9d966e644405bf19b536adf7743d  vo.access.egi.eu     True       CESGA
-916506ac136741c28e4326975eef0bff  vo.emso-eric.eu      True       CESGA
-b1d2ef2cc2284c57bcde21cf4ab141e3  vo.nextgeoss.eu      True       CESGA
-eb7ff20e603d471cb731bdb83a95a2b5  fedcloud.egi.eu      True       CESGA
-fcaf23d103c1485694e7494a59ee5f09  vo.notebooks.egi.eu  True       CESGA
+eval "$(fedcloud show-project-id --site CESGA --vo vo.notebooks.egi.eu)
```
Comment (andrea-manzi) [inline content/en/providers/notebooks/operations/_index.md:59]:
```suggestion
eval "$(fedcloud openstack show-project-id --site CESGA --vo vo.notebooks.egi.eu)"
```

File: content/en/users/cloud-compute/ec3/apps/htc/_index.md [hunk 2]
```diff
@@ -10,12 +10,12 @@ description: >
...
 
 We will build a torque cluster on one of the EGI Cloud providers using EC3.
-Create a directory to store EC3 configuration and init it with `egicli`:
+Create a directory to store EC3 configuration and init it with `fedcloud`:
 
 ```shell
 mkdir -p torque
 cd torque
-egicli endpoint ec3 --site <your site> --project-id <project_id>
+fedcloud ec3 init --site <your site> --project-id <project_id>
```
Comment (andrea-manzi) [inline content/en/users/cloud-compute/ec3/apps/htc/_index.md:18]:
```suggestion
fedcloud ec3 --site <your site> --project-id <project_id>
```

File: content/en/users/cloud-compute/ec3/cli/_index.md [hunk 3]
```diff
@@ -66,70 +66,26 @@ your cluster:
...
---------------------------------  ----------------  ---------  ------
-3a8e9d966e644405bf19b536adf7743d  vo.access.egi.eu  True       CESGA
-```
-
-Using the project id and the site name, you can create the authorisation files
-needed for ec3:
-
-```shell
-egicli endpoint ec3 --site CESGA --project-id 3a8e9d966e644405bf19b536adf7743d
+fedcloud ec3 --site CESGA --project-id vo.access.egi.eu
```
Comment (andrea-manzi) [inline content/en/users/cloud-compute/ec3/cli/_index.md:75]:
```suggestion
fedcloud endpoint ec3 --site CESGA --project-id vo.access.egi.eu
```

File: content/en/users/cloud-compute/ec3/cli/_index.md [hunk 4]
```diff
@@ -315,12 +271,12 @@ debug*       up   infinite      1   idle wn1
 Once you are done with the cluster and want to destroy it, you can use the
 `destroy` command. If your cluster was created more than one hour ago, your
 credentials to access the site will be expired and need to refreshed first with
-`egicli endpoint ec3-refresh`:
+`fedcloud ec3 refresh`:
```
Comment (andrea-manzi) [inline content/en/users/cloud-compute/ec3/cli/_index.md:274]:
```suggestion
`fedcloud endpoint ec3 refresh`:
```

File: content/en/users/cloud-compute/faq/_index.md [hunk 5]
```diff
@@ -62,28 +62,33 @@ sites.
 ### How can I get the list of the EGI Cloud providers?
 
 The list of certified providers is available in [GOCDB](https://goc.egi.eu). The
-`egicli endpoint list` command can help you to get that list:
+`fedcloud endpoint list` command can help you to get that list:
```
Comment (andrea-manzi) [inline content/en/users/cloud-compute/faq/_index.md:65]:
```suggestion
`fedcloud site list` command can help you to get that list:
```

File: content/en/users/cloud-compute/faq/_index.md [hunk 6]
```diff
@@ -62,28 +62,33 @@ sites.
...
-IISAS-FedCloud      org.openstack.nova  https://nova.ui.savba.sk:5000/v3/
-UNIV-LILLE          org.openstack.nova  https://thor.univ-lille.fr:5000/v3
-INFN-PADOVA-STACK   org.openstack.nova  https://egi-cloud.pd.infn.it:443/v3
-CYFRONET-CLOUD      org.openstack.nova  https://api.cloud.cyfronet.pl:5000/v3/
-SCAI                org.openstack.nova  https://fc.scai.fraunhofer.de:5000/v3
-CESNET-MCC          org.openstack.nova  https://identity.cloud.muni.cz/v3
-INFN-CATANIA-STACK  org.openstack.nova  https://stack-server.ct.infn.it:35357/v3
-CESGA               org.openstack.nova  https://fedcloud-osservices.egi.cesga.es:5000/v3
-100IT               org.openstack.nova  https://cloud-egi.100percentit.com:5000/v3/
+$ fedcloud endpoint list
```
Comment (andrea-manzi) [inline content/en/users/cloud-compute/faq/_index.md:68]: ### 
```suggestion
$ fedcloud site list
```

File: content/en/users/cloud-compute/ec3/apps/htc/_index.md (hunk 2 above)
Comment (tdviet) [inline content/en/users/cloud-compute/ec3/apps/htc/_index.md:18]:
```suggestion
fedcloud ec3 init --site <your site> --vo <your vo>
```

File: content/en/users/cloud-compute/auth/_index.md [hunk 7]
```diff
@@ -132,29 +135,15 @@ Once you know which project to use, you can use your regular openstack cli
...
 ```shell
-# Export OIDC env
-export CHECKIN_CLIENT_ID=<CLIENT_ID>
-export CHECKIN_CLIENT_SECRET=<CLIENT_SECRET>
-export CHECKIN_REFRESH_TOKEN=<REFRESH_TOKEN>
-# EGI site
-export EGI_SITE=<NAME_OF_THE_SITE>
-eval "$(egicli endpoint token)"
+export OS_TOKEN=$(fedcloud openstack --site <NAME_OF_SITE> --vo <NAME_OF_VO> \
+                  token issue -c id -f value)
```
Comment (tdviet) [inline content/en/users/cloud-compute/auth/_index.md:146]: Wait for fixing issue 33 https://github.com/tdviet/fedcloudclient/issues/33 to make this command work, a new version should be available on PyPi after weekend

File: content/en/providers/notebooks/operations/_index.md [hunk 8]
```diff
@@ -52,39 +52,25 @@ cd <new provider>/terraform
...
 ```shell
-$ egicli endpoint projects --site CESGA
-id                                Name                 enabled    site
---------------------------------  -------------------  ---------  ------
-3a8e9d966e644405bf19b536adf7743d  vo.access.egi.eu     True       CESGA
-916506ac136741c28e4326975eef0bff  vo.emso-eric.eu      True       CESGA
-b1d2ef2cc2284c57bcde21cf4ab141e3  vo.nextgeoss.eu      True       CESGA
-eb7ff20e603d471cb731bdb83a95a2b5  fedcloud.egi.eu      True       CESGA
-fcaf23d103c1485694e7494a59ee5f09  vo.notebooks.egi.eu  True       CESGA
+eval "$(fedcloud openstack show-project-id --site CESGA --vo vo.notebooks.egi.eu)"
```
Comment (tdviet) [inline content/en/providers/notebooks/operations/_index.md:59]:
```suggestion
eval "$(fedcloud site show-project-id --site CESGA --vo vo.notebooks.egi.eu)"
```

File: content/en/users/cloud-compute/ec3/cli/_index.md (hunk 3 above)
Comment (tdviet) [inline content/en/users/cloud-compute/ec3/cli/_index.md:75]:
```suggestion
fedcloud ec3  init--site CESGA --vo vo.access.egi.eu
```

**Review by tdviet — COMMENTED** [review]
A new version of fedcloudclient, with issue 33 fixed will be uploaded to PyPi after weekend. Documentation updated during next week.

File: content/en/users/cloud-compute/auth/_index.md (hunk 7 above)
Comment (tdviet) [reply content/en/users/cloud-compute/auth/_index.md:146]: Fixed.

Comment (enolfc) [reply content/en/users/cloud-compute/auth/_index.md:146]: nice, is this available in PyPi already?

Comment (tdviet) [reply content/en/users/cloud-compute/auth/_index.md:146]: Yes, fedcloudclient 1.2.4
https://pypi.org/project/fedcloudclient/

Comment (tdviet) [reply content/en/users/cloud-compute/auth/_index.md:146]: In addition, for documentation:
- info about ec3 commands added: https://fedcloudclient.fedcloud.eu/usage.html#fedcloud-ec3-commands
- new guide for using fedcloudclient in scripts added: https://fedcloudclient.fedcloud.eu/scripts.html

Comment (enolfc) [reply content/en/users/cloud-compute/auth/_index.md:146]: Thanks! @andrea-manzi, please check if PR looks ok now so we can merge

**Review by andrea-manzi — APPROVED** [review]
I did some tests with the new version released and all seems good!

**Pull request merged**